docs/src/routes/index.svelte

```diff
@@ -0,0 +1,43 @@
+<script>
+  import { Card, Button, H1 } from 'attractions';
+  import { BookOpenIcon, GithubIcon } from 'svelte-feather-icons';
+  import InfoTiles from 'src/containers/home/info-tiles.svelte';
+</script>
+
```

Comment:
Let's also add a `<svelte:head>` with all the OpenGraph meta for link previews?

Reply:
Yeah, that's all in the coming

Reply:
This PR or in the future?

Reply:
Future, this one's ready for review